### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by ggrimaudDTS**

### CODE REVIEW

File: utah-design-system-website/src/react/components/websiteContent/library/components/containers/Card/CardDocumentation.jsx [hunk 1]
```diff
@@ -33,10 +62,55 @@ export function CardDocumentation() {
...
+            </div>
+          </div>
+        `}
+      />
 
       <StaticExample
         title="Action Cards"
-        renderedExample={<LightBox image={actionCardsScreenshot} alt="Action Cards" className="flex-3up-gap" />}
+        renderedExample={(
+          <div className="flex flex-col gap">
```
Comment (AustinHaws-Utah) [inline utah-design-system-website/src/react/components/websiteContent/library/components/containers/Card/CardDocumentation.jsx:93]: Should the cursor be a hand when these are hovered?

File: utah-design-system-website/src/react/components/websiteContent/library/components/containers/Card/CardDocumentation.jsx [hunk 2]
```diff
@@ -33,10 +62,55 @@ export function CardDocumentation() {
...
+              <div className="action-card__title">
+                <h3>Utah Header and Footer</h3>
+                <span className="button--icon button--icon-right"><span className="utds-icon-before-arrow-right" aria-hidden="true" /></span>
+              </div>
+              <div className="action-card__body">
+                Start using the official header and footer.
+              </div>
+            </div>
+          </div>
+        )}
```
Comment (AustinHaws-Utah) [inline utah-design-system-website/src/react/components/websiteContent/library/components/containers/Card/CardDocumentation.jsx:113]: The quicktip has a list with a single item which seems like it probably shouldn't be a list? Though the other ones are lists with more than one item... so maybe it's fine?

File: utah-design-system-website/src/react/components/websiteContent/library/components/containers/Card/CardDocumentation.jsx [hunk 3]
```diff
@@ -48,10 +122,90 @@ export function CardDocumentation() {
...
+              <div className=" p-spacing-xl">
+                <div className="card__title">Dark Sky Parks in Utah</div>
+                <p>Sed ut perspiciatis unde omnis iste natus error sit
+                  voluptatem accusantium doloremque laudantium,
+                  totam rem aperiam, eaque ipsa quae ab illo inventore
+                  veritatis et quasi architecto beatae vitae dicta sunt
+                  explicabo voluptatem accusantium doloremque.
+                </p>
+                <div className="flex justify-end">
+                  <button type="button" className="button">Contact Us</button>
```
Comment (AustinHaws-Utah) [inline utah-design-system-website/src/react/components/websiteContent/library/components/containers/Card/CardDocumentation.jsx:181]: These buttons have the wrong background color when hovered? they disappear into the background.

File: utah-design-system-website/src/react/components/websiteContent/library/components/containers/Card/CardDocumentation.jsx (hunk 1 above)
Comment (ggrimaudDTS) [reply utah-design-system-website/src/react/components/websiteContent/library/components/containers/Card/CardDocumentation.jsx:93]: Yes

File: utah-design-system-website/src/react/components/websiteContent/library/components/containers/Card/CardDocumentation.jsx (hunk 2 above)
Comment (ggrimaudDTS) [reply utah-design-system-website/src/react/components/websiteContent/library/components/containers/Card/CardDocumentation.jsx:113]: Leave it

File: utah-design-system-website/src/react/components/websiteContent/library/components/containers/Card/CardDocumentation.jsx (hunk 3 above)
Comment (ggrimaudDTS) [reply utah-design-system-website/src/react/components/websiteContent/library/components/containers/Card/CardDocumentation.jsx:181]: Will fix

**Pull request merged**